src/include/duckdb/planner/expression_binder.hpp

```diff
@@ -175,6 +181,10 @@ class ExpressionBinder {
 	ClientContext &context;
 	optional_ptr<ExpressionBinder> stored_binder;
 	vector<BoundColumnReferenceInfo> bound_columns;
+
+	//! Constant aliases for the UNNEST function
+	const string UNNEST_FUNCTION_ALIAS = "unnest";
```

Comment:
Maybe this could be a function instead, e.g. `static bool IsUnnest(const string &function_name)`